src/pages/home/Organizers.tsx

```diff
@@ -1,64 +1,68 @@
 import * as React from "react";
-import donavon2 from "../../assets/donavon@2x.png";
-import harry2 from "../../assets/harry@2x.png";
-import jenn2 from "../../assets/jenn@2x.png";
-import matthew2 from "../../assets/matthew@2x.png";
+import { shuffle } from "lodash";
+import donavon from "../../assets/donavon@2x.png";
+import harry from "../../assets/harry@2x.png";
+import jenn from "../../assets/jenn@2x.png";
+import matthew from "../../assets/matthew@2x.png";
 import Section from "../../components/Section";
 import TextGroup from "../../components/TextGroup";
 
+type Organizer = {
+  twitter: string;
+  image: string;
+  name: string;
+};
+const Organizers: Organizer[] = shuffle([
+  {
+    image: donavon,
+    name: "Donavon West",
+    twitter: "donavon"
+  },
+  {
+    image: harry,
+    name: "Harry Wolff",
+    twitter: "hswolff"
+  },
+  {
+    image: jenn,
+    name: "Jenn Creighton",
+    twitter: "gurlcode"
+  },
+  {
+    image: matthew,
+    name: "Matthew Gerstman",
+    twitter: "MatthewGerstman"
+  }
+]);
+
 export default () => {
   return (
     <Section id="organizers" className="organizers last-section">
       <TextGroup>
         <h2 className="centered-text">The Organizers</h2>
       </TextGroup>
       <ul className="organizers-ul">
-        <li>
-          <a
-            href="https://twitter.com/donavon"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            <div>
-              <img src={donavon2} />
-              <p>Donavon West</p>
-            </div>
-          </a>
-        </li>
-        <li>
-          <a
-            href="https://twitter.com/hswolff"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            <div>
-              <img src={harry2} /> <p>Harry Wolff</p>
-            </div>
-          </a>
-        </li>
-        <li>
-          <a
-            href="https://twitter.com/gurlcode"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            <div>
-              <img src={jenn2} /> <p>Jenn Creighton</p>
-            </div>
-          </a>
-        </li>
-        <li>
-          <a
-            href="https://twitter.com/MatthewGerstman"
-            rel="noopener noreferrer"
-            target="_blank"
-          >
-            <div>
-              <img src={matthew2} /> <p>Matthew Gerstman</p>
-            </div>
-          </a>
-        </li>
+        {Organizers.map((organizer) => (
+          <Organizer {...organizer} key={organizer.name} />
```

Comment:
in theory, 2 organizers _could_ have the same name. a better (and guaranteed unique) `key` would be their twitter handle. Just sayin' 🤷‍♀️